scripts/generateAssetData/baseAssets.ts

```diff
@@ -1,12 +1,16 @@
 import * as caip from '@shapeshiftoss/caip'
+import type { WithRequiredProperty } from 'lib/types'
 
 import type { Asset } from '../../src/lib/asset-service'
 
-export const ethereum: Asset = {
+type AssetWithNetworkName = WithRequiredProperty<Asset, 'networkName'>
```

Comment:
want to avoid the extra util type and just use an intersection?
`type AssetWithNetworkName = Asset & { networkName: string }`

Reply:
241731470135fc99c561169a8b4f59cdb7a8f378